Foody/Onboarding/Register/RegisterViewController.swift

```diff
@@ -6,24 +6,146 @@
 //
 
 import UIKit
+import Firebase
+import FirebaseAuth
+import FirebaseCore
 
 class RegisterViewController: UIViewController {
-
+    
+    @IBOutlet weak var usernameText: UITextField!
+    @IBOutlet weak var emailText: UITextField!
+    @IBOutlet weak var passwordText: UITextField!
+    @IBOutlet weak var dateText: UITextField!
+    @IBOutlet weak var signUpClicked: UIButton!
+    @IBOutlet weak var passwordLabel: UILabel!
+    
+    let datePicker = UIDatePicker()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        
+        signUpClicked.layer.cornerRadius = 10.0
+        
+        usernameText.placeholder = "Username"
+        emailText.placeholder = "Email"
+        passwordText.placeholder = "Password"
+        dateText.placeholder = "Date of Birth"
+        passwordLabel.text = "Şifre en az 6 karakterli en az bir büyük harf ve bir rakam içermelidir."
+        
+        datePicker.addTarget(self, action: #selector(dateChange(datePicker:)), for: UIControl.Event.valueChanged)
+        datePicker.frame.size = CGSize(width: 0, height: 300)
+        datePicker.preferredDatePickerStyle = .wheels
+        
+        datePicker.maximumDate = Date()
+        createDatePicker()
     }
     
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    @objc func dateChange(datePicker: UIDatePicker) {
+        dateText.text = formatDate(date: datePicker.date)
+    }
+    
+    func createDatePicker() {
+        let toolbar = UIToolbar()
+        toolbar.sizeToFit()
+        
+        let doneButton = UIBarButtonItem(barButtonSystemItem: .done, target: nil, action: #selector(doneTapped))
+        toolbar.setItems([doneButton], animated: true)
+        dateText.inputAccessoryView = toolbar
+        
+        dateText.inputView = datePicker
+        datePicker.datePickerMode = .date
+    }
+    
+    @objc func doneTapped () {
+        let formatter = DateFormatter()
```

Comment:
Remove this formater. After clicked done it will be different format that you expect (dd/MM/yyyy) 
```suggestion
        dateText.text = formatDate(date: datePicker.date)
```